lib/services/web-view-service.ts

```diff
@@ -0,0 +1,80 @@
+///<reference path="../.d.ts"/>
+"use strict";
+import semver = require("semver");
+
+export class WebViewService implements IWebViewService {
+	
+	constructor(private $errors: IErrors,
+		private $pluginsService: IPluginsService,
+		private $project: Project.IProject,
+		private $projectConstants: Project.IProjectConstants,
+		private $options: IOptions) { }
+		
+	public get minSupportedVersion() {
+		return "4.0.0";	
+	} 
+	
+	public validateFrameworkVersion(): void {
+		if(semver.lt(this.$project.projectData.FrameworkVersion, this.minSupportedVersion)) {
+			this.$errors.failWithoutHelp(`Web view releated commands are supported only for Cordova ${this.minSupportedVersion} or greater.`);
```

Comment:
Please improve this error message. Once I got it, I am left helpless, because I have no directions how to fix the issue. I suggest to add something similar to "To change Cordova version to the minimum supported one, use $ appbuilder mobileframework set ${this.minSupportedVersion}". @ikoevska, opinion?